README.md

```diff
@@ -2,208 +2,394 @@
+within the `if` statement, such as `favoriteFruit`, does not exist outside of
+that block.
+
+## Gotcha's to Be Aware Of
+
+Now that we've covered the main three scopes, there are two gotcha's you should
+be aware of moving forward. Both have to do with how you declare a variable.
 
-## The global gotcha
+All of our examples thus far have either used `const` or `let` to declare a
+variable. For best practices, we recommend you always use one of the two.
+However, you may encounter legacy code that uses `var` or doesn't use a keyword
+at all. In such cases, those variables do not follow the scope rules we just
+learned about.
 
-In a perfect world, you'd always remember to declare new variables with `const`
-and `let`, and you'd never run into any weird scoping issues. However, it's
-inevitable that at some point you're going to forget the `const` or `let` and
-accidentally do something like:
+## The `var` Gotcha
+
+If you declare a variable with **`var`**, that variable **does not get block
+scoped**. Let's demonstrate this with a modified version of our `if` statement
+example from before:
 
 ```js
-firstName = "Ada";
+const animal = "capybara";
+
+if (animal === "capybara") {
+  var favoriteFruit = "yuzu";
+  console.log("A capybara's favorite bath fruit is: ", favoriteFruit); // => A capybara's favorite bath fruit is: yuzu
+}
+
+console.log(favoriteFruit); // => yuzu
 ```
 
-Variables created without a `const`, `let`, or `var` keyword are **always
-globally-scoped**, regardless of where they sit in your code. If you create one
-inside of a block, it's still available globally:
+Because it was defined with `var`, we are able to access `favoriteFruit` outside
+of the `if` statement it was defined in.
+
+However, `var` does **get function scoped**. For example, let's wrap our `if`
+statement inside a function:
 
 ```js
-if (true) {
-  lastName = "Lovelace";
+const animal = "capybara";
+
+function logAnimalFruit() {
+  if (animal === "capybara") {
+    var favoriteFruit = "yuzu";
+    console.log("A capybara's favorite bath fruit is: ", favoriteFruit); // => A capybara's favorite bath fruit is: yuzu
+  }
+  return favoriteFruit;
 }
 
-lastName;
-// => "Lovelace"
+logAnimalFruit(); // => yuzu
+console.log(favoriteFruit); // => ReferenceError: favoriteFruit is not defined
 ```
 
-If you create one inside of a function — wait for it — it's still available
-globally:
+As we saw before, `favoriteFruit` does not get scoped to the `if` statement's
+block. So, when we try to access the variable inside the `logAnimalFruit`
+function that it's nested within, it is available. However, when we try to
+access it _outside_ of the function like we did before, we are now unable to
+access it.
+
+In sum, `var` gets scoped to its nearest parent function. If it's not declared
+within a function at all, it gets scoped globally.
+
+Confusing, isn't it? This is all the more reason to **never use `var`** when
```

Comment:
```suggestion
If this is confusing, don't worry - this is one reason JavaScript has been updated to include `let` and `const` instead of `var` and is all the more reason to **never use `var`** when
```